debian.master/config/annotations

```diff
@@ -9152,6 +9152,7 @@ CONFIG_PAGE_SIZE_LESS_THAN_256KB                policy<{'amd64': 'y', 'arm64': '
 CONFIG_PAGE_SIZE_LESS_THAN_64KB                 policy<{'amd64': 'y', 'arm64': 'y', 'arm64-generic-64k': '-', 'armhf': 'y', 'riscv64': 'y', 's390x': 'y'}>
 CONFIG_PAGE_TABLE_CHECK                         policy<{'amd64': 'n', 'arm64': 'n', 'riscv64': 'n'}>
 CONFIG_PAGE_TABLE_ISOLATION                     policy<{'amd64': 'y'}>
+CONFIG_PAHOLE_HAS_LANG_EXCLUDE                  policy<{'amd64': 'y', 'arm64': 'y', 'armhf': 'y', 'ppc64el': 'y', 'riscv64': 'y', 's390x': 'y'}>
```

Comment:
I don't understand - why does reverting this specific thing help us with the new gcc?
Can we not revert the whole thing to enable rust with the new Debian?

Reply:
Ubuntu's kernel uses rustc 1.68.2.  But, Debian has rustc 1.63.0+dfsg1-2.  If rustc updates the version, then the config has to be updated again.
https://github.com/endlessm/linux/commit/4a4400fe74d3a698a41cbc377a8a9756ac56d35a#diff-58801bea3f8e9a2d4b3123d542d16bbef45442929b5b6cc17da4919caf19479dL11035